### PR TITLE
Update UG for pay and owe command

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, the pay and owe command in the user guide does not follow the same style as the other commands

Let's
*Create a standardized format for all the commands 
*Edit the pay and owe command to follow the format

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -202,37 +202,43 @@ Examples:
...
+![payResult.png](images/payResult.png)
 
 <box type="important" header="##### Constraints">
 
-1. The index refers to the index number shown in the displayed student list.
-2. The index **must be a positive integer** 1, 2, 3, …​
-3. Hours paid field should be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
+* The **INDEX** refers to the index number shown in the displayed student list.
+* The **INDEX must be a positive integer** 1, 2, 3, …​
+* **HOURS_PAID** should be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
```
Comment (TY1Fan) [inline docs/UserGuide.md:218]: Might be good to add in a tip for using edit to change the paid amount if made a mistake. Similar to the tip for owe command below.

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -202,37 +202,49 @@ Examples:
...
-Example: 
-* `owe 1 hr/1.5` updates the tuition fee owed by the 2nd student in the list.
+**Output:**
 ![oweResult.png](images/oweResult.png)
 
 <box type="important" header="##### Constraints">
-    Hours owed by a student must be a positive multiple of 0.5.
+    
+* **HOURS_OWED** must be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
+
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:242]: Should this not have the index constraints descriptions?

**Review by Soumyadip-cmd — COMMENTED** [review]
Minor changes

**Pull request merged**